fdbserver/DataDistribution.actor.cpp

```diff
@@ -2569,6 +2578,186 @@ ACTOR Future<Void> waitUntilHealthy(DDTeamCollection* self, double extraDelay =
 	}
 }
 
+// This function is very expensive to call. Use with caution.
+ACTOR Future<Void> printSnapshotTeamsInfo(Reference<DDTeamCollection> self, std::string tcIdString) {
+	state DatabaseConfiguration configuration;
+	state std::map<UID, Reference<TCServerInfo>> server_info;
+	state ServerStatusMap server_status;
```

Comment:
We don't need AsyncMap for this. 
We only need to record the set of <UID, ServerStatus>. 

std::set<UID, ServerStatus> is better than ServerStatusMap

Reply:
Done.